pg-index-health-spring-boot-demo/src/test/java/io/github/mfvanek/pg/index/health/demo/PgIndexHealthSpringBootDemoApplicationTest.java

```diff
@@ -32,4 +33,11 @@ void contextLoads() {
             .isNotNull()
             .isInstanceOf(WebMvcConfigurer.class);
     }
+
+    @Test
+    void jdbcQueryTimeoutFromProperties() {
```

Comment:
We also need a test that fails with timeout. Use pg_sleep https://postgrespro.ru/docs/postgresql/16/functions-datetime#FUNCTIONS-DATETIME-DELAY